client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -689,4 +691,33 @@ public void getTemplateAsync(GetIndexTemplatesRequest getIndexTemplatesRequest,
         restHighLevelClient.performRequestAsyncAndParseEntity(getIndexTemplatesRequest, RequestConverters::getTemplates,
             options, GetIndexTemplatesResponse::fromXContent, listener, emptySet());
     }
+
+    /**
+     * Puts an stored script using the Scripting API.
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/6.2/modules-scripting.html"> Scripting API
+     * on elastic.co</a>
+     * @param putStoredScriptRequest the request
+     * @param options the request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
+     * @return the response
+     * @throws IOException in case there is a problem sending the request or parsing back the response
+     */
+    public PutStoredScriptResponse putStoredScript(PutStoredScriptRequest putStoredScriptRequest,
```

Comment:
this API call doesn't belong to any category and is described at https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/put_script.json - and it looks like it has to go to main `RestHighLevelClient` ( e.g. in comparison to https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/indices.create.json that is for category `indices`)

Could you please move those methods to `RestHighLevelClient` ?

Reply:
should I also remove the stored parts from the class names?

Reply:
no those have to stay as they are as it's already existing classes that people have been using with the transport client.